packages/x-data-grid-premium/src/tests/dataSourceAggregation.DataGridPremium.test.tsx

```diff
@@ -104,7 +104,9 @@ describeSkipIf(isJSDOM)('<DataGridPremium /> - Data source aggregation', () => {
       expect(fetchRowsSpy.callCount).to.be.greaterThan(0);
     });
     await user.click(within(getColumnHeaderCell(0)).getByLabelText('id column menu'));
-    expect(await screen.findByLabelText('Aggregation')).not.to.equal(null);
+    await waitFor(async () => {
```

Comment:
It will be strange if this helps. 🤔 
Essentially, we are wrapping it another waitFor (findBy is already async). 🤔

Reply:
Yes, `findBy` = `getBy` + `waitFor` https://testing-library.com/docs/dom-testing-library/api-async#findby-queries

We don't seem to need `expect` here at all since `findBy` will throw if element is not found

Reply:
Opened https://github.com/mui/mui-x/pull/17311